### DISCUSSION TITLE
Erro ao gravar a versão 1.0.16

### DISCUSSION THREAD

**Discussion by HudsonBroetto** [uncodead/BrewUNO#77] [Q&A]
Olá,
Sou usuário do BrewUNO a um tempo, mas meu ESP8266 apresentou defeito, e precisei comprar um novo.
Mas não consegui gravar o novo ESP8266, está apresentando o seguinte erro:

"test offset :  0 0x0
case ok
test offset :  3145728 0x300000
case ok
Traceback (most recent call last):
  File "download_panel_info.py", line 448, in OnButton_StartButton
  File "download_panel_info.py", line 421, in start_func
  File "download_panel_info.py", line 703, in flash_config
UnicodeDecodeError: 'gb2312' codec can't decode bytes in position 24-25: illegal multibyte sequence"

Utilizei o flash_download_tool_3.8.5 para efetuar a gravação.
Tentei gravar 2 Wemos D1 mini, o erro ocorreu nos dois.

Alguém passou por isso?
Alguém possui alguma versão anterior do firmware?

Muito obrigado.

**Comment by uncodead** [comment]
O problema persiste? Já testou em outras versões?

**Reply by HudsonBroetto** [reply]
Bom dia,
Não consegui gravar a versão 1.0.16.
Mas a versão 1.0.15 funcionou perfeitamente.

Muito obrigado!
